src/test/java/com/zufar/icedlatte/order/converter/OrderDtoConverterTest.java

```diff
@@ -0,0 +1,104 @@
+package com.zufar.icedlatte.order.converter;
+
+import com.zufar.icedlatte.openapi.dto.OrderItemResponseDto;
+import com.zufar.icedlatte.openapi.dto.OrderResponseDto;
+import com.zufar.icedlatte.order.entity.Order;
+import com.zufar.icedlatte.order.entity.OrderItem;
+import com.zufar.icedlatte.order.stub.OrderDtoTestStub;
+import com.zufar.icedlatte.product.converter.ProductInfoDtoConverter;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static com.zufar.icedlatte.order.stub.OrderDtoTestStub.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+@SpringBootTest(classes = {OrderDtoConverterTest.Config.class})
+class OrderDtoConverterTest {
+
+    @Autowired
+    ProductInfoDtoConverter productInfoDtoConverter;
+    @Autowired
+    OrderItemDtoConverter orderItemDtoConverter;
+    @Autowired
+    OrderDtoConverter orderDtoConverter;
+
+    @Configuration
+    public static class Config {
+
+        @Bean
+        public OrderDtoConverter orderDtoConverter() {
+            return Mappers.getMapper(OrderDtoConverter.class);
+        }
+        @Bean
+        public OrderItemDtoConverter orderItemDtoConverter() {
+            return Mappers.getMapper(OrderItemDtoConverter.class);
+        }
+        @Bean
+        public ProductInfoDtoConverter productInfoDtoConverter() {
+            return Mappers.getMapper(ProductInfoDtoConverter.class);
+        }
+    }
+
+    @Test
+    @DisplayName("toOrderEntity should convert OrderRequestDto to Order entity with complete information")
+    void shouldConvertOrderRequestDtoToOrderEntity() {
+        var orderRequestDto = OrderDtoTestStub.createOrderRequestDto();
+        Order order = orderDtoConverter.toOrderEntity(orderRequestDto);
+
+        assertEquals(orderRequestDto.getDeliveryCost(), order.getDeliveryCost());
+        assertEquals(orderRequestDto.getTaxCost(), order.getTaxCost());
+        assertEquals(orderRequestDto.getDeliveryInfo(), order.getDeliveryInfo());
+        assertEquals(orderRequestDto.getRecipientName(), order.getRecipientName());
+        assertEquals(orderRequestDto.getRecipientSurname(), order.getRecipientSurname());
+        assertEquals(orderRequestDto.getEmail(), order.getEmail());
+        assertEquals(orderRequestDto.getPhoneNumber(), order.getPhoneNumber());
```

Comment:
moreover I suggest you to completely remove those tests for mappers as they are not following the test design practices + we don't need it since it does not make so much sence to test predefined map struct mappers. We dont use the custom impl , so if u really want to write some additional tests , please cover the most advanced things in mapping